Classes/Authentication/Common/HDWallet/Data/HDWalletSetupDataController.swift

```diff
@@ -20,18 +20,52 @@ final class HDWalletSetupDataController {
     var eventHandler: ((Event) -> Void)?
 
     var items: [HDWalletItemViewModel] = []
```

Comment:
Items can be modified from the outside. Maybe it will be a good idea to make it `private(set)`